contracts/util/TellerCommon.sol

```diff
@@ -36,7 +54,14 @@ library TellerCommon {
         uint256 requestTime;
     }
 
-    // Consensus response object for accruing interest
+    /**
+        @notice Consensus response object for accruing interest
+        @param signer The waller address of the signer validating the interest request of the lender
+        @param consensusAddress The wallet address of the signer submitted the interest response of the submitted request
```

Comment:
Missing a word between `signer` and `submitted` I think

Reply:
Fixed